ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -153,10 +155,17 @@ def install_mcg_only_cluster(self):
 
         """
         logger.info("Install MCG ONLY cluster via UI")
-        self.do_click(self.dep_loc["advanced_deployment"])
+        if self.ocp_version == "4.9":
+            self.do_click(self.dep_loc["advanced_deployment"])
         self.do_click(self.dep_loc["expand_advanced_mode"], enable_screenshot=True)
-        self.do_click(self.dep_loc["mcg_only_option"], enable_screenshot=True)
-        self.do_click(self.dep_loc["next"], enable_screenshot=True)
+        if self.ocp_version == "4.9":
+            self.do_click(self.dep_loc["mcg_only_option"], enable_screenshot=True)
+        elif self.ocp_version == "4.10":
+            self.do_click(self.dep_loc["mcg_only_option_4_10"], enable_screenshot=True)
```

Comment:
Do we have a locator change in OCP 4.10 when compared to OCP 4.9? If not, the existing locator should still work no matter what is the position of that element on the same page. Please check.

Reply:
The locators on OCP4.10 and OCP4.9 are different